kubernetes_platform/python/test/unit/test_tolerations.py

```diff
@@ -0,0 +1,193 @@
+
+        @dsl.pipeline
+        def my_pipeline():
+            task = comp()
+            kubernetes.add_tolerations(
+                task,
+                [
+                    V1Toleration(
+                        key='key1',
+                        operator='Equal',
+                        value='value1',
+                    ),
+                    V1Toleration(
+                        key='key2',
+                        operator='Equal',
+                        value='value2',
+                    ),
+                ],
+            )
+
+        assert json_format.MessageToDict(my_pipeline.platform_spec) == {
+            'platforms': {
+                'kubernetes': {
+                    'deploymentSpec': {
+                        'executors': {
+                            'exec-comp': {
+                                'tolerations': [
+                                    {
+                                        'key': 'key1',
+                                        'operator': 'Equal',
+                                        'value': 'value1',
+                                    },
+                                    {
+                                        'key': 'key2',
+                                        'operator': 'Equal',
+                                        'value': 'value2',
+                                    },
+                                ]
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+    def test_respects_other_configuration(self):
+
+        @dsl.pipeline
+        def my_pipeline():
+            task = comp()
+            kubernetes.use_secret_as_volume(
+                task, secret_name='my-secret', mount_path='/mnt/my_vol')
+            kubernetes.add_tolerations(
+                task,
+                [
+                    V1Toleration(
+                        key='key1',
+                        operator='Equal',
+                        value='value1',
+                    ),
+                ],
+            )
+
+        assert json_format.MessageToDict(my_pipeline.platform_spec) == {
+            'platforms': {
+                'kubernetes': {
+                    'deploymentSpec': {
+                        'executors': {
+                            'exec-comp': {
+                                'tolerations': [{
+                                    'key': 'key1',
+                                    'operator': 'Equal',
+                                    'value': 'value1',
+                                },],
+                                'secretAsVolume': [{
+                                    'secretName': 'my-secret',
+                                    'mountPath': '/mnt/my_vol',
+                                },],
+                            },
+                        }
+                    }
+                }
+            }
+        }
+
+
+@dsl.component
+def comp():
```

Comment:
is this required?

Reply:
Yup -- since it's referenced by each of the tests, e.g. [here](https://github.com/kubeflow/pipelines/pull/10471/files/002bfe96903ec1766151a9d6dada7f38303be481#diff-d0cc953dcb63eaf93dcc71bef9e5b13e48feecdcca9836476b6c60bf99b5b869R108).